### ISSUE TITLE
Sanity check: Verify that libreoffice produced the output PDF file

### ISSUE THREAD

**Issue by deeplow** [freedomofpress/dangerzone#370]
In some occasions LibreOffice (e.g. [this file](https://github.com/freedomofpress/dangerzone-test-set/blob/main/all_documents/encryptedGPG.odt)) fails to convert but exits with zero:

```
libreoffice --headless --convert-to pdf:writer_pdf_Export --outdir /tmp encryptedGPG.odt
```

This is a file that asks for as password and that's why it fails.

**Comment by deeplow** [comment]
This is not a big deal because the tool that runs on the file afterwards will fail. But this could add extra debug time since we may think it's an issue with the second tool instead of the first one.

**Comment by apyrgio** [comment]
Fixed in bc83341d2a21551736b62d80974d6eba6bc75420.
